OpenRA.Mods.Common/Traits/Turreted.cs

```diff
@@ -62,48 +67,95 @@ public Turreted(ActorInitializer init, TurretedInfo info)
 		{
 			this.info = info;
 			TurretFacing = GetInitialTurretFacing(init, info.InitialFacing);
+
+			if (info.RandomRealign)
+				randomFacing = init.Self.World.SharedRandom.Next(0, 256);
```

Comment:
Why?

Sounds overly complicated to me. It should work exactly like how `WithInfantryBody` picks an idle animation, and moving/attacking should switch back to face-where-you're-going/attacking behaviour. Generals and beyond did that, using turret realigning as an idle anim.